Cargo.toml

```diff
@@ -34,6 +34,7 @@ crossbeam = { version = "0.8.4" }
 dirs = { version = "5.0.0" }
 drop_bomb = { version = "0.1.5" }
 env_logger = { version = "0.11.0" }
+etcetera = { version = "0.8.0" }
```

Comment:
@konstin suggested this crate, and it does what we want across Windows and Linux / macOS, but open to other suggestions.